client/src/components/WrappedTicket/index.jsx

```diff
@@ -1,65 +1,27 @@
-import React, { Component } from 'react';
+import React from 'react';
 import { Link, Route } from 'react-router-dom'
 import './style.css'
-class WrappedTicket extends Component {
-    state = {
-        wrappedTickets: [],
-    }
-    componentDidMount() {
-        this.setState({
-            wrappedTickets: [{
-                id: 1,
-                email: 'jamalat@gmail.com',
-                description: 'This is a sample ticket with some sample description',
-                date: 'April 9'
-                , subject: "subject"
-            },
-            {
-                id: 2,
-                email: 'jamalat@gmail.com',
-                description: 'This is a sample ticket with some sample description',
-                date: 'April 9'
-                , subject: "subject"
-            }, {
-                id: 3,
-                email: 'jamalat@gmail.com',
-                description: 'This is a sample ticket with some sample description',
-                date: 'April 9'
-                , subject: "subject"
-            }, {
-                id: 4,
-                email: 'jamalat@gmail.com',
-                description: 'This is a sample ticket with some sample description',
-                date: 'April 9'
-                , subject: "subject"
-            }]
-        })
-    }
-    render() {
-        if (this.state.wrappedTickets) {
-            return (
-                <div className="wrapped-ticket-container">
-                    <div> {this.state.wrappedTickets.map(wrappedticket =>
-                        <Route>
-                            <Link to={`/ticket/${wrappedticket.id}`} className="wrapped-ticket-container__link">
-                                <div className="wrapped-ticket-container__wrappedticket" >
-                                    <div className="wrapped-ticket-container__check-email-container"><input type="checkbox" className="wrapped-ticket-container__checkbox" />
-                                        {wrappedticket.email}
-                                    </div>
-                                    <p>{wrappedticket.subject}</p>
-                                    <p className="wrapped-ticket-container__wrappedticket__ticket-descreption">{wrappedticket.description.substring(0, 49)}...</p>
-                                    <p className="wrapped-ticket-container__date"> {wrappedticket.date} </p>
-                                </div></Link>
-                        </Route>
+const WrappedTicket = (props) => {
+    console.log(props)
+    return (
+        <div className="wrapped-ticket-container">
+            <div> {props.wrappedTickets ? props.wrappedTickets.map(wrappedticket =>
+                <Route>
```

Comment:
@Jamalat-shamallakh I think a `<Route` that has no `path` is basically just a `<Fragment`, it just always renders, so you could replace the `<Route` with a fragment or div.